guides/rql.md

```diff
@@ -139,30 +140,47 @@ a particular occurrence evaluate to NULL.
 | `item.last_muted_timestamp` | Timestamp the item was last muted
 | `item.last_occurrence_id` | ID of the most recent occurrence
 | `item.last_occurrence_timestamp` | Timestamp of the most recent occurrence
-| `item.total_occurrences` | Number of occurrences since last resolved
 | `item.last_modified_by` | ID of the user who last modified this item
-| `item.status` | Status (active, resolved, muted)
-| `item.timestamp` | When the error occurred, as a unix timestamp
-| `item.level` | Level (critical, error, warning, info, debug)
-| `item.language` | The name of the language your code is written in
-| `item.code_version` | The version of the application code
-| `item.context` | An identifier for which part of your application the error came from
-| `item.fingerprint` | A string controlling how the occurrence is grouped
-| `item.title` | The occurrence title
-| `item.uuid` | A string that uniquely identifies the occurrence
-| `item.resolved_in_version` | Revision the item was last resolved in
-| `item.notifier.name` | Name of the library that sent the item
-| `item.notifier.library` | The version string of the library that sent the item
-| `client.javascript.browser` | Raw user agent string
+| `item.level` | Item level (50=critical, 40=error, 30=warning, 20=info, 10=debug)
+| `item.resolved_in_version` | Revision the item was last marked as resolved in
+| `item.status` | Status (as an integer: 1=active, 2=resolved, 3=muted)
+| `item.title` | Computed title
+| `item.total_occurrences` | Number of occurrences since last resolved
```

Comment:
This isn't super clear...perhaps "The number of occurrences since the last time this item was resolved"